common/build/eslint-config-fluid/strict-ts.json

```diff
@@ -0,0 +1,1440 @@
+    "unicorn/no-hex-escape": [
+      "error"
+    ],
+    "unicorn/no-instanceof-array": [
+      "error"
+    ],
+    "unicorn/no-invalid-remove-event-listener": [
+      "error"
+    ],
+    "unicorn/no-keyword-prefix": [
+      "off"
+    ],
+    "unicorn/no-lonely-if": [
+      "error"
+    ],
+    "no-nested-ternary": [
+      "off"
+    ],
+    "unicorn/no-nested-ternary": [
+      "error"
+    ],
+    "unicorn/no-new-array": [
+      "error"
+    ],
+    "unicorn/no-new-buffer": [
+      "error"
+    ],
+    "unicorn/no-null": [
+      "error"
+    ],
+    "unicorn/no-object-as-default-parameter": [
+      "error"
+    ],
+    "unicorn/no-process-exit": [
+      "error"
+    ],
+    "unicorn/no-static-only-class": [
+      "error"
+    ],
+    "unicorn/no-thenable": [
+      "error"
+    ],
+    "unicorn/no-this-assignment": [
+      "error"
+    ],
+    "unicorn/no-unreadable-array-destructuring": [
+      "error"
+    ],
+    "unicorn/no-unsafe-regex": [
+      "off"
+    ],
+    "unicorn/no-unused-properties": [
+      "off"
+    ],
+    "unicorn/no-useless-fallback-in-spread": [
+      "error"
+    ],
+    "unicorn/no-useless-length-check": [
+      "error"
+    ],
+    "unicorn/no-useless-promise-resolve-reject": [
+      "error"
+    ],
+    "unicorn/no-useless-spread": [
+      "error"
+    ],
+    "unicorn/no-useless-undefined": [
+      "error"
+    ],
+    "unicorn/no-zero-fractions": [
+      "error"
+    ],
+    "unicorn/number-literal-case": [
+      "error"
+    ],
+    "unicorn/numeric-separators-style": [
+      "error"
+    ],
+    "unicorn/prefer-add-event-listener": [
+      "error"
+    ],
+    "unicorn/prefer-array-find": [
+      "error"
+    ],
+    "unicorn/prefer-array-flat": [
+      "error"
+    ],
+    "unicorn/prefer-array-flat-map": [
+      "error"
+    ],
+    "unicorn/prefer-array-index-of": [
+      "error"
+    ],
+    "unicorn/prefer-array-some": [
+      "error"
+    ],
+    "unicorn/prefer-at": [
+      "off"
+    ],
+    "unicorn/prefer-code-point": [
+      "error"
+    ],
+    "unicorn/prefer-date-now": [
+      "error"
+    ],
+    "unicorn/prefer-default-parameters": [
+      "error"
+    ],
+    "unicorn/prefer-dom-node-append": [
+      "error"
+    ],
+    "unicorn/prefer-dom-node-dataset": [
+      "error"
+    ],
+    "unicorn/prefer-dom-node-remove": [
+      "error"
+    ],
+    "unicorn/prefer-dom-node-text-content": [
+      "error"
+    ],
+    "unicorn/prefer-export-from": [
+      "error"
+    ],
+    "unicorn/prefer-includes": [
+      "error"
+    ],
+    "unicorn/prefer-json-parse-buffer": [
+      "error"
+    ],
+    "unicorn/prefer-keyboard-event-key": [
+      "error"
+    ],
+    "unicorn/prefer-math-trunc": [
+      "error"
+    ],
+    "unicorn/prefer-modern-dom-apis": [
+      "error"
+    ],
+    "unicorn/prefer-module": [
+      "error"
+    ],
+    "unicorn/prefer-negative-index": [
+      "error"
+    ],
+    "unicorn/prefer-node-protocol": [
+      "error"
+    ],
```

Comment:
I'm planning to disable this, because the node protocol support was only added in 12.20/14.13.1, and earlier versions just blow up with the syntax.

Reply:
I'll enable [import/no-nodejs-modules](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-nodejs-modules.md) as a replacement. Using node built-ins should always be explicit.